### PR TITLE
Updated the kissr package to try and convert any times found in a rep…

### PR DESCRIPTION
**Pull request by yosemsweet**
…ort to be from the timezone KISSR__KISSMETRICS_CONFIGURED_TIMEZONE_ZONENAME (specified via ENV VARS). Then return the actual times in UTC.

### CODE REVIEW

**Comment by yosemsweet** [comment]
Fix for Issue #6

**Comment by yosemsweet** [comment]
To change the timezones for times returned by KissMetrics to UTC, simply specify the appropriate timezone in your env vars using KISSR__KISSMETRICS_CONFIGURED_TIMEZONE_ZONENAME.

For instance: KISSR__KISSMETRICS_CONFIGURED_TIMEZONE_ZONENAME ="America/Vancouver"

File: R/utilities.R [hunk 1]
```diff
@@ -101,3 +101,24 @@ loadPages <- function (object, url) {
   }
   return(result)
 }
+
+# We only support a specific format returned by KissMetrics, so we'll use
+# lubridate::fast_strptime instead of parse_date_time.
+# Take a character vector, and if we think it is a time that can be parsed by
+# format we'll return a vector of times, if it can't be converted (any of the
+# non-NA elements fail to parse) we return the original vector
+try_convert_time <- function(char_vector, formats = "%Y-%m-%d %H:%M:%S") {
```
Comment (mrtns) [inline R/utilities.R:110]: Sucks that KM's timestamp resolution is only to the second? (For most user behaviour tracking, milliseconds with at least three digits is preferred)

File: R/utilities.R [hunk 2]
```diff
@@ -101,3 +101,24 @@ loadPages <- function (object, url) {
...
+  result <- char_vector
+
+  timezone <- Sys.getenv("FROM_TIMEZONE")
```
Comment (mrtns) [inline R/utilities.R]: `KISSMETRICS_CONFIGURED_TIMEZONE`? and probably postfixed with `_ZONENAME` or `_ZONECODE`?

Comment (mrtns) [inline R/utilities.R]: And maybe even `KISSR__KISSMETRICS_CONFIGURED_TIMEZONE_ZONENAME`

**Comment by mrtns** [comment]
:+1:

File: R/utilities.R (hunk 1 above)
Comment (yosemsweet) [inline R/utilities.R:110]: Yeah, KM stores things internally as UNIX timestamps. Even when we do the full data export via the S3 bucket we'll still only have to the second fidelity :(

Interestingly enough when we do that export kissmetrics will also write events in the relevant json files based of the order they were received, not the timestamp.

So if the method we use for sending the events to KM is the same we can know the order the events occurred, however for any events that are sent asynchronously the best we can do is the timestamp.

File: R/utilities.R (hunk 2 above)
Comment (yosemsweet) [inline R/utilities.R]: KISSR__KISSMETRICS_CONFIGURED_TIMEZONE_ZONENAME  works for me

**Pull request merged**